conf/defaults.ini

```diff
@@ -1927,3 +1927,6 @@ domain = grafana-dev.net
 # Should UI tests fail when console log/warn/erroring?
 # Does not affect the result when running on CI - only for allowing devs to choose this behaviour locally
 fail_tests_on_console = true
+# Whether or not to enable the MSW mock API, which intercepts requests and returns mock data
+# Should only be used for local development or demo purposes
+mock_api = false
```

Comment:
Actual usage would be for devs to override this with `custom.ini` and set to true, so their local UI dev env can run MSW for new feature development. Then we can scaffold out new API behaviour before its completed, write tests etc...